README.md

```diff
@@ -1,11 +1,11 @@
-# Runnable Templates
+# Templates
 
-This repo is the basis for creating your own Runnable templates for Reactr, Atmo, and Suborbital Compute. TypeScript/AssemblyScript, Rust, and Swift are supported.
+This repo is the basis for creating your own Runnable templates for E2Core and Suborbital Compute. TypeScript/AssemblyScript, Rust, and Swift are supported.
 
 To get started, fork this repo into your own organization. It must be public.
 
-Next, clone your fork and edit `templates/assemblyscript/src/custom.ts` and `templates/assemblyscript/src/lib.ts` to change the template that will be used to create new Runnables. If you're a Flight Deck user, `lib.ts` is what will be used for all new functions when your users access the code editor.
+Next, clone your fork and edit `templates/assemblyscript/src/custom.ts` and `templates/assemblyscript/src/lib.ts` to change the template that will be used to create new modules. If you're a Flight Deck user, `lib.ts` is what will be used for all new functions when your users access the code editor.
```

Comment:
@cohix Do you have a quick suggestion for the Flight Deck reference here?

Reply:
Flight Deck == SE2 (I cannot believe those are still around lmao)